DigitalOffice/Tests/ProjectServiceTests/Validators/WorkersProjectIdsValidatorTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿using FluentValidation;
+using FluentValidation.TestHelper;
+using LT.DigitalOffice.ProjectService.Models;
+using LT.DigitalOffice.ProjectService.Validators;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.ProjectServiceUnitTests.Validators
+{
+    class WorkersProjectIdsValidatorTests
+    {
+        private List<Guid> workerIds;
+
+        private IValidator<WorkersIdsInProjectRequest> validator;
+        private WorkersIdsInProjectRequest workersProjectRequest;
+
+        [SetUp]
```

Comment:
This can be done only once, because our validator not changing anything.

Reply:
Fixed